src/controllers/customercontroller.js

```diff
@@ -140,22 +140,18 @@ export default class CustomerController extends OrderController
         });
     }
 
-    /**
```

Comment:
Since the order/customer view is also updated by the score controller, I have changed it to a 'public' function. I hope that's not a problem/should be implemented differently.

Reply:
That should be fine, I think.